### DISCUSSION TITLE
StackView is not available .

### DISCUSSION THREAD

**Discussion by naser09** [arkivanov/Decompose#414] [Q&A] [answered]
i am using decompose latest 2.0.0-beta-01 and following the docs for test implementation . but for some reason StackView is not available . 
```
var body: some View {
        StackView(
            stackValue: StateValue(root.stack),
            getTitle: {
                switch $0 {
                case is RootComponentChild.ListChild: "List"
                case is RootComponentChild.DetailsChild: "Details"
                default: ""
                }
            },
            onBack: counters.onBackClicked,
            childContent: {
                switch $0 {
                case let child as RootComponentChild.ListChild: ListView(child.component)
                case let child as RootComponentChild.DetailsChild: DetailsView(child.component)
                default: EmptyView()
                }
            }
        )
    }
```

**Comment by arkivanov** [comment, marked as answer]
It has to be copied from the sample code. Please see the docs: https://arkivanov.github.io/Decompose/getting-started/quick-start/#what-is-stackview

**Reply by naser09** [reply]
thank you so much for your replay . i already did that but and my apps working fine .anyway in the docs https://arkivanov.github.io/Decompose/getting-started/quick-start/#child-stack-with-swiftui 
```
onBack: counters.onBackClicked,
```
 its clear that its not counter but  rootComponent .i know that the sample apps has counter component but quick start guide have rootComponent , List and Detail component. and also
i needed to change `getTitle: {}`
switch code to 
```
switch $0 {
       case is RootComponentChild.ListChild: return "List"
      case is RootComponentChild.DetailsChild: return "Details"
       default: return ""
   }
```
because it was showing that it missing return something ..

**Reply by arkivanov** [reply]
Thank you so much for your feedback. I updated that doc just today and those were mistakes indeed. Fixed now!
